test/system/src/main/java/io/pravega/test/system/framework/Utils.java

```diff
@@ -158,6 +159,7 @@ public static ClientConfig buildClientConfig(URI controllerUri) {
                                // auth
                                .credentials(new DefaultCredentials("1111_aaaa", "admin"))
                                .controllerURI(controllerUri)
+                               .rawclientTimeout(Duration.ofSeconds(3600))
```

Comment:
Why so long? It seems like for tests we'd want it rather short so we catch any stalls before the test times out.